tcrudge/decorators.py

```diff
@@ -13,11 +13,7 @@ async def func(self, *args, **kw):
             auth = await self.is_auth()
             if auth:
                 roles = await self.get_roles()
-                valid_permission = False
-                for r in roles:
-                    if r in items:
-                        valid_permission = True
-                        break
+                valid_permission = any(r in items for r in roles)
```

Comment:
Function `perm_roles.wrap.func` refactored with the following changes:

- Use any() instead of for loop ([use-any](https://docs.sourcery.ai/refactorings/use-any/))